src/components/ProductCard/ProductCard.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { FC, MouseEvent, useCallback } from 'react'
+import { useTranslation } from 'react-i18next'
+
+import Image from 'next/image'
+import Link from 'next/link'
+
+import likeIcon from 'img/svg/heart-icon-orange.svg'
+import basketIcon from 'img/svg/to-basket.svg'
+import { IMenuName } from 'interface/catalogInterface'
+import { ICardProduct } from 'interface/productInterface'
+import { useGetColorsQuery } from 'store/services/colors'
+import { Routes } from 'utils/constants/routes'
+import { getBlurImage } from 'utils/helpers/getBlurImage'
+import { productColors } from 'utils/helpers/productColors'
+
+import { ColorCard } from 'components/ColorCard'
+
+import styles from './ProductCard.module.scss'
+
+type Props = {
+  product: ICardProduct
+}
+
+export const ProductCard: FC<Props> = ({ product }) => {
+  const { data, isSuccess } = useGetColorsQuery()
+  const colors = isSuccess ? productColors(product.colors, data) : []
+
+  const getBlur = useCallback((img: string) => getBlurImage(img), [])
+
+  const { t } = useTranslation()
+  const categories = t('menu', { returnObjects: true }) as IMenuName[]
+
+  const handlerClickLike = (e: MouseEvent) => {
+    e.preventDefault()
```

Comment:
move it to utils and use every where, and add TODO or comment, whan is this.

Reply:
deleted, will be used in the future